src/types/composer.ts

```diff
@@ -201,101 +247,164 @@ export type AlgokitComposerParams = {
   defaultValidityWindow?: number
 }
 
+/** AlgoKit Composer helps you compose and execute transactions as a transaction group.
+ *
+ * Note: this class is a new Beta feature and may be subject to change.
+ */
 export default class AlgokitComposer {
   /** Map of txid to ABI method */
   private txnMethodMap: Map<string, algosdk.ABIMethod> = new Map()
 
   /** Transactions that have not yet been composed */
   private txns: Txn[] = []
 
-  /** The underlying AtomicTransactionComposer */
-  atc: algosdk.AtomicTransactionComposer
```

Comment:
Turns out we didn't need this as state and by creating it on the fly we can call `buildGroup` multiple times without error.

Reply:
I had this as a property in the event of calling `buildGroup` prior to `execute`. It's important to preserve the same ATC between the two calls otherwise the suggestedParams may change. I realize this means we should probably expose the ATC status and add a check in execute

Reply:
And if you want to build multiple times, I'd propose a `rebuild` method

Reply:
Feel free to try and refactor it back, I pulled it out to solve for a case where I needed to call buildGroup preemptively and then it caused the underlying atc to error

Reply:
[61f0428](https://github.com/algorandfoundation/algokit-utils-ts/pull/261/commits/61f04288fbe820105bf89a69cf0d43211a47884e) should prevent errors when calling build multiple times while also adding a `rebuild` method for explicitly rebuilding with new sp